### PR TITLE
Feat/user story 1 habit list

### PR DESCRIPTION
**Pull request by FelixJncs**

### CODE REVIEW

File: Data/HabitsDB.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    id: 9,
+    name: "Nail picking or skin picking",
+    reason: "Reason: A nervous or anxious habit",
+    feeling: "Feeling: Embarrassed or ashamed",
+  },
+  {
+    id: 10,
+    name: "Interrupting others",
+    reason: "Reason: Impatience or desire to be heard",
+    feeling: "Feeling: Annoyed or frustrated when not being heard",
```
Comment (djfarly) [inline Data/HabitsDB.js]: I'd suggest not including repeated strings in your data :)

```suggestion
    reason: "Impatience or desire to be heard",
    feeling: "Annoyed or frustrated when not being heard",
```

File: components /HabitCard.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    h3 {
+      margin: 0;
+    }
+  `;
+
+  return (
+    <StyledHabitCard>
+      <h3>{habit.name}</h3>
+      <p>{habit.reason}</p>
+      <p>{habit.feeling}</p>
```
Comment (djfarly) [inline components /HabitCard.js]: Instead add them here:

```suggestion
      <p>Reason: {habit.reason}</p>
      <p>Feeling: {habit.feeling}</p>
```

File: components /HabitCard.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  const StyledHabitCard = styled.li`
+    background-color: beige;
+    border-radius: 2rem;
+    padding: 1rem;
+    box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
+    h3 {
+      margin: 0;
+    }
+  `;
+
```
Comment (djfarly) [inline components /HabitCard.js]: Please define your styled components outside of your actual component function:

```suggestion
import styled from "styled-components";

const StyledHabitCard = styled.li`
  background-color: beige;
  border-radius: 2rem;
  padding: 1rem;
  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
  h3 {
    margin: 0;
  }
`;

export default function HabitCard({ habit }) {
```

File: components /HabitList.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import HabitCard from "./HabitCard";
+
+export default function HabitList({ habits }) {
+  const StyledHabitList = styled.ul`
+    list-style-type: none;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: center;
+  `;
```
Comment (djfarly) [inline components /HabitList.js]: same here

```suggestion
const StyledHabitList = styled.ul`
  list-style-type: none;
  display: flex;
  flex-direction: column;
  gap: 1rem;
  justify-content: center;
`;
  
export default function HabitList({ habits }) {
```

File: pages/index.js [hunk 5]
```diff
@@ -1,7 +1,19 @@
...
+import styled from "styled-components";
+
+export default function Home() {
+  const StyledHabitListContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 2rem;
+  `;
```
Comment (djfarly) [inline pages/index.js]:
```suggestion
import styled from "styled-components";

const StyledHabitListContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  padding: 2rem;
`;

export default function Home() {
```

**Review by djfarly — CHANGES_REQUESTED** [review]
Hej Felix,
thank you for your first PR!

I added some notes below, but rest looks great :)

Please fix the things mentioned below and keep going 👍

File: Data/HabitsDB.js (hunk 1 above)
Comment (djfarly) [reply Data/HabitsDB.js]: please remove those string from all entries ;)

**Review by mbosselmann — COMMENTED** [review]
Hi Felix!
LGTM! 🦊 Nothing else to remark. 💪

**Pull request merged**